src/random/urandom.nim

```diff
@@ -65,33 +65,35 @@ template urandomImpl(): stmt {.immediate.} =
   when defined(windows):
     if cryptProv == 0:
       urandomInit()
-    
+
     let success = CryptGenRandom(cryptProv, DWORD(size), addr result[0])
     if success == 0:
       raise newException(OSError, "Call to CryptGenRandom failed")
-  
+
```

Comment:
Could you perhaps go through all source files in master and remove all trailing whitespaces then? What's the point in keeping them? I can create another PR for that if necessary.